docs/source/using/con_static-ip-overview.adoc

```diff
@@ -0,0 +1,7 @@
+Most hypervisors do not support extending the lease time when the IP is assigned using DHCP.
+This might lead to a new IP being assigned to the VM after a restart as it will conflict with the security certificates generated for the old IP.
+This will make {project} completely unusable.
```

Comment:
we might want to include the necessary fix if encountered? `until a minishift delete and minishift start` has been run to setup a completely new instance`.

`To prevent this, {project} includes ...`